src/application/components/Explorer/postMessageAuthHelpers.ts

```diff
@@ -0,0 +1,75 @@
+import { IncomingMessageEvent, postMessageToEmbed } from "./postMessageHelpers";
+
+export const SET_PARTIAL_AUTHENTICATION_TOKEN_FOR_PARENT =
+  "SetPartialAuthenticationTokenForParent";
+export const EXPLORER_LISTENING_FOR_PARTIAL_TOKEN =
+  "ExplorerListeningForPartialToken";
+export const PARTIAL_AUTHENTICATION_TOKEN_RESPONSE =
+  "PartialAuthenticationTokenResponse";
+
```

Comment:
most of this is copy pasted from studio: https://github.com/mdg-private/studio-ui/blob/main/src/app/embedHelpers/useListenForAuthResponse.tsx